### PR TITLE
feat: fix zk init in rollup mode

### PR DESCRIPTION
**Pull request by toni-calvin**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.
- [ ] Linkcheck has been run via `zk linkcheck`.

### CODE REVIEW

File: yarn.lock
Comment (ilitteri) [inline yarn.lock:1]: Let's revert these changes

**Review by ilitteri — CHANGES_REQUESTED** [review]
The PR looks good, well done! We can merge after reverting the `yarn.lock` changes.

File: yarn.lock
Comment (toni-calvin) [reply yarn.lock:1]: Done!

**Review by ilitteri — APPROVED** [review]
LGTM
